py3status/modules/xsel.py

```diff
@@ -32,13 +38,25 @@ class Py3status:
     """
     # available configuration parameters
     cache_timeout = 0.5
-    command = 'xsel -o'
+    command = 'xsel -output'
+    click_command = 'xsel --clear --clipboard'
     format = '{selection}'
     max_size = 15
     symmetric = True
+    log_file = None
+
+    selection_cache = None
 
     def xsel(self):
-        selection = self.py3.command_output(self.command)
+        selection = self.py3.command_output(self.command).strip()
+
+        if self.log_file and selection and selection != self.selection_cache:
+            with open(self.log_file, "a") as f:
+                datetime = time.strftime("%Y-%m-%d %H:%M:%S")
+                f.write("{}\n{}\n".format(datetime, selection))
+        self.selection_cache = selection
+
+        selection = re.sub(r"\s+", " ", selection)  # merge whitespace into one
```

Comment:
`selection = ' '.join(selection.split())` works? See if we can avoid `import re`. I have this PR too.